### ISSUE TITLE
building the lastest on gitub

### ISSUE THREAD

**Issue by mightydanp** [GregTechCE/GregTech#489]
when i run gradlew buil i get this error in cmd 
Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.

**Comment by Archengius** [comment]
1) it's not a error, it's a warning
2) Gradle 5.0 is not going be used before 1.13

**Comment by mightydanp** [comment]
ok, let me rephrase it. when I run the build it doesn't build anything it downloads the dependencies then stops
